Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -130,6 +130,14 @@ lemma PosSemidef.eigenvalues_nonneg [DecidableEq n] [DecidableEq 𝕜] {A : Matr
     (hA : Matrix.PosSemidef A) (i : n) : 0 ≤ hA.1.eigenvalues i :=
   (hA.re_dotProduct_nonneg _).trans_eq (hA.1.eigenvalues_eq _).symm
 
+lemma eigenvalues_conjTranspose_mul_self_nonneg (A : Matrix m n 𝕜) [DecidableEq n] [DecidableEq 𝕜] :
```

Comment:
Please merge master, these `DecidableEq` arguments will now be rejected by the linter thanks to #7315.